src/routes/+page.svelte

```diff
@@ -160,15 +187,22 @@
 		margin-bottom: 0.5rem;
 	}
 
-	input {
+	input{
 		width: 100%;
-	}
-
-	input,
-	button {
 		font-size: 1rem;
 		padding: 0.5rem 1rem;
 		margin-bottom: 1rem;
 		box-sizing: border-box;
 	}
+
+	table, th, td {
+		border: 1px solid black;
+		border-collapse: collapse;
+		padding-left: 5px;
+		padding-right: 5px;
+	}
```

Comment:
Two small adjustments suggested to expand the padding some:
<img width="1083" alt="image" src="https://github.com/user-attachments/assets/676e06cb-f659-4fb3-88ba-2f3c4a291401">


```suggestion
	table {
		width: 100%;
	}

	table, th, td {
		border: 1px solid black;
		border-collapse: collapse;
		padding: 5px;
	}
```